WindowsInstall.bat

```diff
@@ -0,0 +1,37 @@
+@echo off
+
+echo This script was successfully tested
+echo on Windows 11 with Python 3.10.6
+pause
+python -c "import os; os.system('wget "https://github.com/cisco/openh264/releases/download/v1.8.0/openh264-1.8.0-win64.dll.bz2"')
```

Comment:
Replace with something that does not depend on wget(which is not part of a standard windows install i believe).
For example:
python -c "import urllib.request; urllib.request.urlretrieve('https://github.com/cisco/openh264/releases/download/v1.8.0/openh264-1.8.0-win64.dll.bz2', 'openh264-1.8.0-win64.dll.bz2')"